src/app/pages/basics/TrafficLayerExample.js

```diff
@@ -0,0 +1,42 @@
+/* global google */
+import {
+  default as React,
```

Comment:
I'm not sure this import notation should be encouraged. What's the reason you're not doing the normal 
`import React, { Component } from 'react'` here?